REFERENCE.md

```diff
@@ -2377,6 +2377,7 @@ Struct[{
     ],
     Optional['StandardOutput']            => Variant[Enum['inherit','null','tty','journal','kmsg','journal+console','kmsg+console','socket'],Pattern[/\A(file:|append:|truncate:).+$\z/]],
     Optional['StandardError']             => Variant[Enum['inherit','null','tty','journal','kmsg','journal+console','kmsg+console','socket'],Pattern[/\A(file:|append:|truncate:).+$\z/]],
+    Optional['StandardInput']             => Variant[Enum['null','tty','tty-force','tty-fail','data','socket'], Pattern[/\A(file:|fd:).+$\z/]],
```

Comment:
Thanks,
Could you add a few lines to the type_tests to `type_aliases/systemd_unit_service_spec.rb`

Reply:
Somethign to accept `file:/me/to` and disallow `/me/to` in particular I think.